sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -148,15 +154,17 @@ export interface EventHubClientOptions {
   webSocketConstructorOptions?: any;
   /**
    * @property
-   * Retry options for all the operations on the client/producer/consumer.
+   * The retry options for all the operations on the client/producer/consumer.
    * This can be overridden by the retry options set on the producer and consumer.
    */
   retryOptions?: RetryOptions;
 }
 
 /**
- * @class EventHubClient
- * Describes the EventHub client.
+ * @class
+ * The main point of interaction with Azure Event Hubs, the client offers a
+ * connection to a specific Event Hub within the Event Hubs namespace and offers operations
+ * for sending event data, receiving events, and inspecting the connected Event Hub.
```

Comment:
I broke the sentence into two, at the `and offers` point. Also updated to `Azure Event Hubs service`.

Reply:
My suggestion:

> The client is the main point of interaction with Azure Event Hubs service. It offers a ...

Reply:
@jsquire I have no disagreements about the accuracy of the statement. I am looking at it from a perspective of "is this sentence the simplest it can be".